cylc/flow/data_messages_pb2.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
We can't configure per-file ignores at present so we have to stick a comment at the top of it.